### ISSUE TITLE
Debugger deattaches/disconnects instantly after start

### ISSUE THREAD

**Issue by espectro93** [IBM-Blockchain/blockchain-vscode-extension#1050]
Debugger deattaches/disconnects instantly after play

## Description
When I try to debug my smart contract I can build it fine and submit transactions but the debugger disconnects after building with my config. I reproduced all the steps it takes for debugging, but the debugger disconnects after step 5.

`1. Ensure you are connected to the local_fabric runtime and that the local_fabric peer is in development mode. Development mode is indicated by an infinity symbol on a peer, under Nodes in the Local Fabric Ops panel. To toggle development mode, right-click the peer and select Toggle Development Mode.
2. Open your smart contract project in your workspace.
3. Open the debug view in Visual Studio Code using the left-hand navigation bar.
4. Select the Debug Smart Contract configuration by using the dropdown in the upper-left.
5. Package and install the smart contract by clicking the play button.
6. Add breakpoints to the smart contract by clicking on the relevant line numbers in your smart contract files.
7. Click Instantiate in the Local Fabric Ops panel. In the Fabric Gateways panel, you can now right click on transactions to submit them, execution will be paused on any breakpoints you've defined.
8. To submit or evaluate a transaction click the blockchain icon on the debug toolbar.`

After Step 5 it disconnects...

My launch.json: 
`{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "fabric:node",
            "request": "launch",
            "name": "Debug Smart Contract",
            "program": "${workspaceFolder}/index.js"
        }
    ]
}`

Apart from the debugging everything works as expected.

## Existing issues
#777 

## Log File
[3-Local Fabric.log](https://github.com/IBM-Blockchain/blockchain-vscode-extension/files/3272894/3-Local.Fabric.log)

## Your Environment

* IBM Blockchain Platform Extension Version: 1.0.2
* Visual Studio Code version: 1.35.0
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
node v10.15.0
npm 6.4.1
* Operating System and version:
Windows 10 Pro

**Comment by heatherlp** [comment]
@espectro93 This does sound like a problem - are you able to restart the debugging session in order to debug (i.e. pressing play again) to continue?

**Comment by espectro93** [comment]
Yes, I can restart the previous debugging session but after 1 or 2 secs it disconnects again. As in #777 mentioned there was a problem with different node versions, I´ve checked that and my vscode cli uses the same version as my git bash.

**Comment by heatherlp** [comment]
@espectro93 is it language specific? what smart contract language are you using? if you use the `Create Smart Contract Project` command to generate a fresh one in a different language, are you able to debug that?

**Comment by heatherlp** [comment]
@espectro93 please can you also attempt to teardown local_fabric, in case this resolves the problem

**Comment by espectro93** [comment]
@heatherlp I´ve tried it with TypeScript and Javascript with the same result. Transactions from my debug package succeed but breakpoints are´nt hit.

**Comment by heatherlp** [comment]
Hi @espectro93 - we've updated some of the debug code in v1.0.3 of the extension - can you try using that please? let me know how you get on

I'm wondering if this is Windows's specific, like #1077

**Comment by espectro93** [comment]
Hey @heatherlp , tried it again but now it´s not creating the debug package when I press play. The local fabric console output looks like in #1077

**Comment by cazfletch** [comment]
Hi @espectro93. From v1.0.3 debug has changed. When you press the play button it no longer creates the package and installs. This is done when you instantiate or upgrade while debugging. To debug you should do the following

1) make sure the fabric is in dev mode
2) press the play button on the debug panel
3) on the debug toolbar press the blockchain icon
4) click instantiate from the drop down list
5) you can now submit transactions

If you make changes to the smart contract, stop the debug session and then start it again. You don't need to upgrade unless you want to debug the function called on upgrade or instantiate.

**Comment by espectro93** [comment]
Yes I saw the debug icon on the toolbar but it just pops up for a second and I am not able to click it in time. I am gonna try it later on my ubuntu system aswell.

**Comment by cazfletch** [comment]
is this still an issue?

**Comment by espectro93** [comment]
Yes same as before ubuntu works but windows does not

**Comment by cazfletch** [comment]
closing as duplicate of #1077
